tripy/pyproject.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 requires-python = ">= 3.9"
 license = {text = "Apache 2.0"}
 dependencies = [
+  "tensorrt~=10.0",
```

Comment:
Why does it have to be a dependency of both `mlir-tensorrt` and `tripy`?

Reply:
We are now using it directly in `plugin.py` to figure out the major version